src/components/dev-hub/card-badge.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { colorMap, size } from './theme';
+import { P5 } from './text';
+import styled from '@emotion/styled';
+
+const CardBadge = styled('div')`
+    bottom: 0;
+    background-color: ${colorMap.greyDarkThree};
+    display: inline-block;
+    font-family: akzidenz;
+    letter-spacing: 1px;
+    margin: ${size.default};
+    padding: 1px ${size.xsmall};
+    position: absolute;
+    text-transform: uppercase;
+    ${({ color }) => color && `border: 1px solid ${color}`}
+`;
+
+const determineColor = contentType => {
+    switch (contentType) {
+        case 'youtube':
+        case 'twitch':
+            return colorMap.salmon;
+        case 'podcast':
+            return colorMap.yellow;
+        case 'article':
+            return colorMap.teal;
+        default:
+            return null;
+    }
+};
+
+export default ({ contentType, color = '' }) => {
+    return (
+        <CardBadge color={color || determineColor(contentType)}>
+            <P5 collapse>
```

Comment:
```suggestion
            <P5 bold collapse>
```
We actually have a `bold` prop for `P` tags! Using this we can remove the strong tags below